src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/MinMax_PowerFxV1CompatibilityRules.txt

```diff
@@ -0,0 +1,346 @@
+﻿#SETUP: PowerFxV1CompatibilityRules
+
+// With this flag enabled, return type is
+// - always argTypes[0] if all elements are [Date, Time, DateTime, Number, Decimal]
+// - otherwise it's Decimal or Number, depending on NumberIsFloat flag
+
+>> Max(1)
+1
+
+>> Min(1)
+1
+
+>> Max(1,2)
+2
+
+>> Min(1,2)
+1
+
+>> Max(0)
+0
```

Comment:
All tests that are the same with PFxV1 on and off don't need to be here - #1398 is enabling all tests to be run in both combinations. This file can be much shorter than it is today

Reply:
If there are conflicts later, I'll resolve them.